app/services/users/profile.rb

```diff
@@ -137,12 +137,14 @@ def vet360_contact_information
     rescue => e
       error_hash = Users::ExceptionHandler.new(e, 'VAProfile').serialize_error
       scaffold.errors << error_hash
-      log_external_service_error(error_hash)
+      log_external_service_error(error_hash, 'vet360_contact_information')
       nil
     end
 
     # rubocop:disable Metrics/MethodLength
     def mpi_profile
+      return handle_non_loa3_user unless user.loa3?
```

Comment:
This is where we address the 401 Unauthorized external service errors. Spoke with Identity team and they agreed on checking for whether a user is LOA3 before we make the call to not raise the error.